src/controllers/AuthController.js

```diff
@@ -0,0 +1,43 @@
+import jwt from 'jsonwebtoken';
+import helpers from '../helpers';
+import services from '../services';
+import findUser from '../services/findUser';
+
+const { forgotPasswordMessage } = helpers;
+const { sendMail } = services;
+
+/**
+ * Auth controller class
+ * @class
+ */
+export default class AuthController {
```

Comment:
Is a class really necessary here?

Reply:
I thought it would be good to group the other auth related controllers in this class. For example, the controller for verifying a user's account could also come into this class, likewise signup and login, since they all relate to auth. Or do you think I should just make the `forgotPassword` method a function and export it like that?

Reply:
I'm of the opinion that exporting functions here is a better option as we don't have any class properties we're using and won't be extending this class. At the end of the day, writing classes still falls back to `function ClassName(){}` and `ClassName.prototype.classMethods = function(){}`

Reply:
Alright. I will change it to a function instead. Thanks for the feedback